CanHazFunny/CanHazFunny/JokeService.cs

```diff
@@ -2,13 +2,14 @@
 
 namespace CanHazFunny;
 
-public class JokeService
+public class JokeService : IJokeServiceInterface
```

Comment:
```suggestion
public class JokeService : IJokeService
```
The `I` prefix indicates that it's an interface. The word is unnecessary/redundant